src/Fieldset/README.md

```diff
@@ -0,0 +1,29 @@
+# Fieldset
+
+Wraps many inputs in a border and allows validating them as one form control.
+
+## API
+
+### `dangerIconDescription` (string or element; optional)
```

Comment:
"danger" seems implementation (aka `Bootstrap`)-specific. This feels like it's some kind've "invalidInputDescription" or something #NamingIsHard

Reply:
Why would somebody use an element here?

Reply:
studio-frontend uses an element here because we wrap every translated string in a span that indicates the language of the string with the `lang` attribute.

It's called `dangerIconDescription` because it describes the exclamation icon that only appears when the danger theme is applied.

I'm really just copying what `asInput` is doing. I personally don't like the whole theme thing because I don't see us ever needing an input/fieldset styled with anything other than red or no-red, and it's not clear what happens if you specify multiple themes in the array.

I could completely redo the API for Fieldset, but then it would be inconsistent with asInput. We cannot also redo the API for asInput without making a breaking change release. I opted to make the least changes and inconsistencies and just copied the API for asInput. If we have a clear plan for what the API should be, then I can adopt that here and then we can update asInput to it in a separate PR.

Reply:
Want to note that it could be styled with yellow warning or any of the other bootstrap coloring variant/themes as well even though we're not doing that for SFE right now.